packages/utils-helpers/src/positionalArraySorter.spec.js

```diff
@@ -0,0 +1,179 @@
+import positionalArraySorter from './positionalArraySorter';
+
+test(`Middle`, () => {
```

Comment:
I suppose you ported the tests from the PHP side? (Just asking :-) )

Reply:
Erm... honestly... no. Will check how they compare.